dfu-tutorials/esp8266/esp8266-dfu-v1.0.0.0/platformio.ini

```diff
@@ -0,0 +1,24 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+# renaming the src directory allows this project to also compile with the Arduino IDE
+src_dir = esp8266-dfu
+
+
+[env:huzzah]
+platform = espressif8266
+board = huzzah
+framework = arduino
+
+lib_deps = 
+	blues/Blues Wireless Notecard@^1.3.17
+monitor_speed = 115200
+upload_port = /dev/cu.SLAB_USBtoUART
```

Comment:
Just happened to notice the `upload_port` - won't this change based on the type of OS being used (I think it's different across Windows/Linux/Mac?)

Reply:
yep, I'll remove that before this goes final, it shouldn't be necessary but sometimes PIO doesn't find the right port. Good catch.